index.ejs

```diff
@@ -66,7 +67,7 @@
 <% } %>
 
 <% for (var chunk in htmlWebpackPlugin.files.chunks) { %>
-<script src="<%= htmlWebpackPlugin.files.chunks[chunk].entry %>"></script>
+<script src="<%=prependUrl%><%= htmlWebpackPlugin.files.chunks[chunk].entry %>"></script>
```

Comment:
I don't quite understand the use case of why this is useful.  These chunks and their paths are all handled by webpack.  This seems liked something that `publicPath` could solve.  https://github.com/webpack/docs/wiki/Configuration#outputpublicpath ...  Have a look, please, and let me know what you think.